content/en/docs/concepts/configuration/secret.md

```diff
@@ -241,6 +241,69 @@ using the `-b` option to split long lines.  Conversely Linux users *should* add
 the option `-w 0` to `base64` commands or the pipeline `base64 | tr -d '\n'` if
 `-w` option is not available.
 
+#### Creating a Secret from Generator
+You can also create a Secret from generators and then apply it to create the object on 
+the Apiserver. The generators
+should be specified in a `kustomization.yaml` inside a directory.
+
+For example, to generate a Secret from files `./username.txt` and `./password.txt`
+```shell
```

Comment:
should this be closed?

Reply:
Do you mean remove the secretGenerator from this page?

Reply:
I re-read this page. Because configurations using kustomize/kustomization.yaml are new, I think you should first introduce, or link to a doc that introduces, the concept of  Kustomize's Generators and how this applies to managing Secrets. May want to add something at the beginning of the page.

Reply:
will add some explanation and a link.

Reply:
updated it to
```
Kubectl supports managing objects using Kustomize since 1.14. With this new feature, you can also create a Secret ...
```